ccd-definition/CaseTypeTab.json

```diff
@@ -289,6 +289,16 @@
     "CaseFieldID": "documents_threshold_document",
     "TabFieldDisplayOrder": 9
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "Channel": "CaseWorker",
+    "TabID": "DocumentsTab",
+    "TabLabel": "Documents",
+    "TabDisplayOrder": 6,
+    "CaseFieldID": "c2DocumentBundle",
+    "TabFieldDisplayOrder": 10
```

Comment:
Minor, but don't we need to modify the other fields' order?

Reply:
Again lets see wether this is needed before or after notice of proceedings docs

Reply:
Thanks, otherwise it looks good to me.